mongodb_migrations/config.py

```diff
@@ -34,7 +36,11 @@ def _from_console(self):
         self.arg_parser.add_argument('--port', type=int, metavar='p', default=self.mongo_port,
                                      help="port of MongoDB")
         self.arg_parser.add_argument('--database', metavar='d',
-                                     help="database of MongoDB", default=self.mongo_database)
+                                     help="database of MongoDB", default=self.mongo_database)        
+        self.arg_parser.add_argument('--authuser', metavar='au',
```

Comment:
since `u` is already in use, we can use `-U` here

Reply:
sure :)